style.css

```diff
@@ -28,15 +23,621 @@ ul {
+}
+
+.more a {
+  color: black;
+}
+
+.popular a {
+  font-size: 1.2rem;
+  color: black;
+}
+
+.index-top a {
+  float: right;
+  font-size: 0.8rem;
+  padding-top: 10px;
+  color: #326891;
+}
+
+.index-lists a {
+  color: black;
+  font-size: 0.9rem;
+  line-height: 1.2rem;
+}
+
+.sub-main a {
+  font-size: 1.1rem;
+  font-weight: bold;
+  line-height: 1.6rem;
+}
+
+.sub-sub a {
+  font-size: 0.75rem;
+}
+
+.more p {
+  font-size: 1.3rem;
+  margin-bottom: 0;
+}
+
+button:hover,
+a:hover,
+img:hover {
+  cursor: pointer;
+}
+
+footer a:hover {
+  text-decoration: underline;
+}
+
+.author a:hover {
+  border: none;
+}
+
+.date {
+  font-size: 0.9rem;
+  margin-bottom: 40px;
+}
+
+.article {
+  width: 600px;
+  margin: 0 auto;
+  text-align: left;
+}
+
+.sign-up {
+  text-align: left;
+  display: flex;
+  justify-content: space-between;
+}
+
+.sign-up p {
+  width: 85%;
+  font-size: 1.2rem;
+}
+
+.sign-up button {
+  height: 2rem;
+  padding: 10px 10px;
+  background: white;
+  border: black 1px solid;
+  font-weight: bold;
+}
+
+.sign-up button:hover {
+  color: white;
+  background: black;
+  transition: ease 0.5s;
+}
+
+.article a:hover {
+  text-decoration: none;
+}
+
+.article h2 {
+  margin: 15px 0;
+}
+
+.subscribe {
+  text-align: center;
+  line-height: 1rem;
+  margin-bottom: 1rem;
+  margin-top: 1rem;
+  font-weight: bold;
+}
+
+.subscribe p {
+  font-size: 1rem;
+  margin: 0;
+}
+
+.s-line {
+  content: "";
+  width: 80px;
+  height: 1px;
+  background-color: darkgrey;
+  margin: 15px auto;
+}
+
+.picks {
+  display: flex;
+  flex-direction: column;
+  justify-self: right;
+  align-self: center;
+  justify-items: left;
+  width: 250px;
+  text-align: left;
+}
+
+.picks h5 {
+  margin-bottom: 5px;
+}
+
+.pick {
+  float: right;
+  display: block;
+  max-width: 200px;
+  margin: 10px 0;
+}
+
+.img-pick {
+  width: 80px;
+  height: 80px;
+}
+
+.pick p {
+  font-size: 1rem;
+  line-height: 1rem;
+  align-self: center;
+  margin: 0;
+  margin-left: 5px;
+  color: black;
+}
+
+.card p {
+  font-size: 1.1rem;
+  line-height: 1.3rem;
+}
+
+.pick p:hover {
+  text-decoration: underline;
+}
+
+.helper {
+  content: "";
+  width: 250px;
+}
+
+.inflation {
+  position: relative;
+  width: 945px;
+  margin: 0 auto;
+  text-align: left;
+}
+
+.img-relative {
+  position: relative;
+}
+
+.boxes {
+  margin-top: 80px;
+  position: absolute;
+  top: 1px;
+  left: 1px;
+  z-index: 10;
+}
+
+.grid-helper {
+  margin-top: 130px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 20px;
+}
+
+.l-font {
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+  text-align: left;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.align-left {
+  margin: 15px auto 15px 0;
+  background-color: black;
+}
+
+.s-font {
+  font-size: 0.7rem;
+  line-height: 1.2rem;
+  text-align: left;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+article .links {
+  display: flex;
+}
```

Comment:
### Aim to remain DRY
- [ ] This rule seems to be repeating more than once and makes you appear as if you are repeating yourself. Do you mind merging these ones so as to make the codebase look neat?

**The repeating rules are on lines `#200` & `#423`**